src/components/MetadataPanel.tsx

```diff
@@ -0,0 +1,21 @@
+import { FunctionComponent } from "react";
+import { useMapEditorContext } from "../MapEditorContext";
+import { getCanvasMousePosition } from "../utilities/getCanvasMousePosition";
+
+export const MetadataPanel: FunctionComponent = () => {
+    const { mousePosition, mapCanvasRef, width, height } =
+        useMapEditorContext();
```

Comment:
Nasty formatting. Any way to improve the rules to prevent this kind of thing?

Reply:
longer line length maybe? I'd have to look into it more